sn_node/src/node/bootstrap/join.rs

```diff
@@ -375,6 +372,43 @@ impl<'a> Joiner<'a> {
         }
     }
 
+    // The retry responses may containing different expected ages, or only having
+    // more than 1/3 of elders copies (which blocks the aggregation).
+    // So a restart with new name shall happen when: received 1/3 of elders copies
+    // And with an age of: the lowest expected age
+    fn insert_retry_response(
+        &mut self,
+        sender: Peer,
+        expected_age: u8,
+        elders: BTreeSet<Peer>,
+        elders_count: u8,
+    ) -> Option<u8> {
+        if !elders.contains(&sender) {
+            error!(
+                "Sender {:?} of the retry-response is not part of the elders {:?}",
+                sender, elders
+            );
```

Comment:
We can save a lot of vertical space if you move variables into the format strings:
```suggestion
            error!("Sender {sender:?} of the retry-response is not part of the elders {elders:?}");
```